src/components/Dropdown/Dropdown.js

```diff
@@ -96,14 +96,17 @@ export default class Dropdown extends PureComponent {
       ...other
     } = this.props;
 
-    const children = React.Children.map(this.props.children, child =>
-      React.cloneElement(child, {
-        onClick: (...args) => {
-          child.props.onClick && child.props.onClick(...args);
-          this.handleItemClick(...args);
-        },
-      })
-    );
+    const children = React.Children
+      .toArray(this.props.children)
+      .filter(child => child)
```

Comment:
Instead of creating a new function here, we may probably use `Boolean` (as a function)?